README.md

```diff
@@ -16,17 +19,32 @@ generate-api-client:
   needs: <any prerequisite earlier job>
   runs-on: ubuntu-latest
   steps:
-  - name: Run Hatch Action
-    uses: birdiecare/hatch@v0.0.5 # <-- use current version
-    with:
-      path: <e.g. https://staging.myservice.com/docs.json>
-      name: <Name of your service, e.g. my-service> # the generated client will automatically be called my-service-client
-      token: <your GitHub Registry Token>
+    - uses: actions/checkout@v2 # <-- this one is needed for hatch to access source code
+    - name: Run Hatch action
+      uses: birdiecare/hatch@v0.0.12 # <-- use current version
+      with:
+        path: <e.g. https://staging.myservice.com/docs.json>
+        name: <Name of your service, e.g. my-service> # the generated client will automatically be called my-service-client
+        token: <your GitHub Registry Token>
+```
+
+Running this Action will publish an NPM package called `my-service-client` as a package **inside the repository of the service running the Action**.
+
+### Exposing swagger models
+
+If you want to create a proxy endpoint in `core-api`, you would like to import the class decorated with `@nestjs/swagger` decorators to avoid re-declaring the swagger docs.
+
+In that case, the `birdiecare/hatch` action can scan your `src` folder of the git repo in search of all the files matching the `*.public-models.ts` glob. Other files will be ignored, so make sure you put the swagger classes in that file.
+
+Then in the `core-api` you can import the file from `package/models/filename` nested import, e.g.:
+```ts
+import { IdentifierMapping } from '@birdiecare/rostering-integrations-client/models/identifier-mapping.public-models';
 ```
 
-Running this Action will publish an NPM package called `my-service-client` as a package __inside the repository of the service running the Action__.
+Be aware that the public models files shouldn't have any other imports than `@nestjs/swagger` and `class-validator` ones! Otherwise (especially in case of relative imports) it gonna fail to compile.
```

Comment:
```suggestion
⚠️  Beware that the public models files shouldn't have any other imports than `@nestjs/swagger` and `class-validator` ones! Otherwise (especially in case of relative imports) it gonna fail to compile.
```